dummy-smtp-server/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:latest
```

Comment:
Do we really need this separate Dockerfile for dummy-smtp-server. We can reuse from: https://github.com/LeastAuthority/feedback-api/tree/add-docker-composer/docker/smtp-server